main.go

```diff
@@ -126,5 +129,10 @@ func main() {
 
 	daemonCommand.PersistentFlags().String(f.Service.Deployer.Installer.Configurer.File.Path, "", "Path to values file.")
 
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Notifier.Slack.Channel, "", "Channel to post Slack notifications to.")
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Notifier.Slack.Emoji, ":older_man:", "Emoji to use for Slack notifications.")
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Notifier.Slack.SlackToken, "", "Token to post Slack notifications with.")
+	daemonCommand.PersistentFlags().String(f.Service.Deployer.Notifier.Slack.Username, "draughtsman", "Username to post Slack notifications with.")
```

Comment:
I miss the configmap and secret changes for the Slack specific settings. The channel for instance should be configured in the configmap. The token at least should be inside of a secret. We usually tracked the scheme of the secret in the readme. This should be done here as well.